lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -466,3 +467,57 @@ def upload_ora2_submission_files(
     TASK_LOG.info('%s, Task type: %s, Upload complete.', task_info_string, action_name)
 
     return UPDATE_STATUS_SUCCEEDED
+
+
+def generate_anonymous_ids(_xmodule_instance_args, _entry_id, course_id, task_input, action_name):  # lint-amnesty, pylint: disable=too-many-statements
+    """
+    Generate a 2-column CSV output of user-id, anonymized-user-id
+    """
+    def _log_and_update_progress(info_string, action, step):
```

Comment:
you don't need to pass `info_string` here, it is an inner method & should have access to all the variables.